jdaviz/configs/imviz/tests/test_parser.py

```diff
@@ -330,23 +330,23 @@ def test_parse_hst_drz(self, imviz_app):
         assert_quantity_allclose(tbl[0]['xcenter'], 1488.5 * u.pix)
         assert_quantity_allclose(tbl[0]['ycenter'], 2576 * u.pix)
         sky = tbl[0]['sky_center']
-        assert_allclose(sky.ra.deg, 3.6840882015888323)
-        assert_allclose(sky.dec.deg, 10.802065746813046)
+        assert_allclose(sky.ra.deg, 3.6840882015888323, rtol=1e-5)
+        assert_allclose(sky.dec.deg, 10.802065746813046, rtol=1e-5)
         data_unit = u.electron / u.s
         assert_quantity_allclose(tbl[0]['background'], 0.0014 * data_unit)
         assert_quantity_allclose(tbl[0]['npix'], 645.98998939 * u.pix)
-        assert_quantity_allclose(tbl[0]['aperture_sum'], 81.01186025 * data_unit)
+        assert_quantity_allclose(tbl[0]['aperture_sum'], 78.352207 * data_unit)
```

Comment:
did the underlying file change?  Or why such a large change in the test-values?

Reply:
Instrument teams recalibrate their data from time to time. Even old data might get updated if the pipeline algorithm has changed or some calibration file (like superbias, superdark, etc) is updated. They don't keep the old copy of the data, so we only get the updated version from MAST. Therefore, any test that rely on such data cannot assume the values will not change. We can only check if the change is reasonable or not, when it happens. I think the level of change here is fine. What do you think?

Reply:
p.s. To prove that this change is not caused by this PR, you can also see same failure in your PR. Or if you rerun any of the old CI run on `main`, I guarantee you it will also fail with the same failure.